src/main/java/edu/rpi/legup/ui/ProofEditorPanel.java

```diff
@@ -293,6 +294,17 @@ public JMenuBar getMenuBar() {
 
         edit.add(fitTreeToScreen);
         fitTreeToScreen.addActionListener((ActionEvent) -> this.fitTreeViewToScreen());
+        {
+            GameBoardFacade.getInstance().getHistory().redo();
+        }
```

Comment:
What do lines 297-299 do?

Reply:
no idea

Reply:
I not change there , I think

Reply:
According to GitHub, those three lines were added by you. I'd remove them from the code to be safe if you're not sure what they do.